services/gateway/tests/unit/registerRpcApi.test.js

```diff
@@ -0,0 +1,168 @@
+		expect(response).toEqual('');
+	});
+
+	it('should throw when called with null or undefined', async () => {
+		[null, undefined].forEach(
+			param => expect(() => dropOneSlashAtBeginning(param)).toThrow(),
+		);
+	});
+});
+
+describe('Test curlyBracketsToColon method', () => {
+	it('should return converted string when called with a string having curly braces', async () => {
+		const response = curlyBracketsToColon('url/{param1}/{param2}');
+		expect(response).toEqual('url/:param1/:param2');
+	});
+
+	it('should return original string when called with a string which does not have curly braces', async () => {
+		const response = curlyBracketsToColon('url/param1/param2');
+		expect(response).toEqual('url/param1/param2');
+	});
+
+	it('should return empty string when called with empty string', async () => {
+		const response = curlyBracketsToColon('');
+		expect(response).toEqual('');
+	});
+
+	it('should throw when called with null or undefined', async () => {
+		[null, undefined].forEach(
+			param => expect(() => curlyBracketsToColon(param)).toThrow(),
+		);
+	});
+});
+
+describe('Test convertType method', () => {
+	it('should return number when called to convert string to number', async () => {
+		const response = convertType('123', 'number');
+		expect(typeof response).toEqual('number');
+		expect(response).toEqual(123);
+	});
+
+	it('should return string when called to convert number to string', async () => {
+		const response = convertType(123, 'string');
+		expect(typeof response).toEqual('string');
+		expect(response).toEqual('123');
+	});
+
+	it('should return string when called to convert array to string', async () => {
+		const response = convertType(new Array(...[1, 2, 3]), 'string');
+		expect(typeof response).toEqual('string');
+		expect(response).toEqual('1,2,3');
+	});
+
+	it('should return boolean when called to convert string to boolean', async () => {
+		const response = convertType('true', 'boolean');
+		expect(typeof response).toEqual('boolean');
+		expect(response).toEqual(true);
+	});
+
+	it('should return param type when called for unknown conversion', async () => {
+		const response = convertType(true, 'object');
+		expect(typeof response).toEqual('boolean');
+		expect(response).toEqual(true);
+	});
+
+	it('should return param type when called with null item', async () => {
+		expect(() => convertType(null, 'object')).toThrow();
+	});
+
+	it('should return param type when called with undefined item', async () => {
+		const response = convertType(undefined, 'string');
+		expect(response).toEqual(undefined);
+	});
+
+	it('should return passed item when called with null toType', async () => {
+		const item = 'str';
+		const response = convertType(item, 'string');
+		expect(response).toEqual(item);
+	});
+
+	it('should return passed item when called with undefined toType', async () => {
+		const item = 'str';
+		const response = convertType(item, 'string');
+		expect(response).toEqual(item);
+	});
```

Comment:
These tests doesn't match with the description
```suggestion
	it('should return passed item when called with null toType', async () => {
		const item = 'str';
		const response = convertType(item, 'string');
		expect(response).toEqual(item);
	});

	it('should return passed item when called with undefined toType', async () => {
		const item = 'str';
		const response = convertType(item, 'string');
		expect(response).toEqual(item);
	});
```

Reply:
Applied.